package.json

```diff
@@ -29,6 +29,7 @@
     "nx": "^16.8.1",
     "nx-cloud": "latest",
     "semver": "^7.3.8",
+    "style-dictionary": "^3.9.1",
```

Comment:
This should not be in the root `package.json`. The way you want to add this package to `core` is by running `npm install style-dictionary -w @pine-ds/core`

Reply:
Thanks! I'll make that update.